### PR TITLE
MCR-2955 Friends are outdated and should be able to be wiped out

### PR DESCRIPTION
**Pull request by toKrause**
[Link to jira](https://mycore.atlassian.net/browse/MCR-2955).

### CODE REVIEW

**Comment by rsteph-de** [comment]
Can we extend this issue and add a filter for empty Friends-Properties here:
https://github.com/MyCoRe-Org/mycore/blob/1ca2311523ec98cadeb4a1733dc678032d820e11/mycore-oai/src/main/java/org/mycore/oai/MCROAIIdentify.java#L115

```java
.filter(Predicate.not(String::isBlank))
````
This would allow to override settings with empty properties:
```
MCR.OAIDataProvider.OAI2.Friends.DBThueringen=
```

**Comment by toKrause** [comment]
> Can we extend this issue and add a filter for empty Friends-Properties

That is a good idea, but should probably be a separate issue.

**Comment by rsteph-de** [comment]
Since it's only one additional line of code I would rephrase the issue title
"improve configuration of OAI friends"

**Review by rsteph-de — APPROVED** [review]
